persistence.sh

```diff
@@ -97,6 +97,11 @@ upsert_file() {
 }
 
 save_ansible_vars() {
+	# Prepend bootstrap.log into ceremony.log before github commits and copies to volumes
+	x=$(cat ~/bootstrap.log; cat ${LOG_FILE})
```

Comment:
What's a more descriptive variable name than `x`?

Reply:
xx?

Reply:
jk, good point.  I'll replace that